### ISSUE TITLE
Old acknowledgments missing

### ISSUE THREAD

**Issue by potato4444** [plfa/plfa.github.io#323]
I have noticed some acknowledgements that were in an older version of the book are not in the current version of the book. Presumably they were lost when the acknowledgements list was automated in https://github.com/plfa/plfa.github.io/commit/f0fed89bcb2e464c6739da6c2b28a13c10d1cfcc#diff-e1a867674ed7d1f8142aa6a28f024465

The people I notice were in the old list but not the current list are:
 - Vikraman Choudhury
 - Ben Darwin
 - koo5
 - Anish Tondwalkar

**Comment by wenkokke** [comment]
I wonder why these contributors were listed under the "pull request" section, without ostensibly having made a pull request... I'll have to investigate.

**Comment by wenkokke** [comment]
Out of curiosity, how did you notice this?

**Comment by potato4444** [comment]
> I wonder why these contributors were listed under the "pull request" section, without ostensibly having made a pull request... I'll have to investigate. 

I don't know about the others, but I got on that list by suggesting a fix in an issue that Phil then applied manually. (I'm in the automated list because of a seperate pull request)

 > Out of curiosity, how did you notice this?

Vikraman Choudhury's name stood out to me when I read the book a year or so ago (probably because it was first in the list). I noticed it was absent from the current version.
